### PR TITLE
issue_5 - property_type Lookup Table

### PR DESCRIPTION
**Pull request by egch**

### CODE REVIEW

**Comment by benfante** [comment]
I fixed the build process. Now it creates and populates both schemas (global and tenant).

Please implement the reference to the property_type as a  numberic foreign key.

File: src/main/java/com/yourrents/api/property/Property.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 import java.util.UUID;
 
-public record Property(UUID uuid, String name, String type, String description,
+public record Property(UUID uuid, String name, UUID typeUuid, String description,
```
Comment (benfante) [inline src/main/java/com/yourrents/api/property/Property.java:25]: As Property-PropertyType it is a many-to-one relationship I would like to resolve it eagerly.

So it should be:

```java
public record Property(UUID uuid, String name, PropertyType type, String description,
  // etc.
```

Of course you'll need to adapt the other classes for using it.

**Review by benfante — CHANGES_REQUESTED** [review]
Please resolve the PropertyType in the Property.

**Comment by egch** [comment]
Implemented changes as required. @benfante please review

**Pull request merged**